src/commands/android/utils/common.ts

```diff
@@ -10,6 +10,9 @@ import which from 'which';
 import {symbols} from '../../../utils';
 import {ABI, AVAILABLE_OPTIONS, DEFAULT_CHROME_VERSIONS, DEFAULT_FIREFOX_VERSION, SDK_BINARY_LOCATIONS} from '../constants';
 import {Platform, SdkBinary} from '../interfaces';
+import Logger from '../../../logger';
+import untildify from 'untildify';
+import {execSync} from 'child_process';
```

Comment:
Can you move these imports to the top of file (according to the alphabetical order)? The import in most of the files are divided into two sections: external imports and internal imports.

Reply:
I have done that. However, are we looking at the name of the import or the name of the package for alphabetical order?
For eg. `import {execSync} from 'child_process';` here shall I look at 'e' or 'c'?

Reply:
For external imports, we look at the package name. So, we'd look at the 'c' in the above case.

Idt we have any rule defined for the internal imports yet. The idea here is to have some rule (any rule which makes sense, maybe something suggested by the community) and then follow it throughout the project. And it'd be best if we can document them as well at some place.

Reply:
There is one idea I have about arranging internal imports. We can arrange the import statements in the increasing order of length of the line. The shorter statement above and the longer statement below.

Reply:
We could do that but then the length of import statements may change when making changes to the file in the future for which we'd also need to rearrange the order of import statements again and again to keep up with this rule, which is not ideal.

We could maybe arrange the import statements in the order of the length of filepaths instead, keep the outermost import path at the top and the innermost import path at the bottom. And for the paths of the same length, arrange them alphabetically.

For example, for this file, the internal imports would arrange in the following fashion:
```js
import Logger from '../../../logger';
import {symbols} from '../../../utils';
import {ABI, AVAILABLE_OPTIONS, DEFAULT_CHROME_VERSIONS, DEFAULT_FIREFOX_VERSION, SDK_BINARY_LOCATIONS} from '../constants';
import {Platform, SdkBinary} from '../interfaces';
```
(`logger` is 3 parent-directory up while `constants` is only one parent-directory up.)